### ISSUE TITLE
Issue running checkpoints

### ISSUE THREAD

**Issue by danindiana** [karpathy/char-rnn#192]
Currently at a loss. 

Description of Issue: 

I moved the contents of the last training session out of the char-rnn-master/cv directory to another drive. Then when I wanted to revisit a different a checkpoint with a lower validation loss figure I put said file back in the cv directory, but when I attempted to run the program it spits out the following:

speed@speed-desktop:~/Downloads/char-rnn-master$ th sample.lua lm_lstm_epoch5.05_1.1291.t7 -gpuid -0 -length 10000
using CUDA on GPU 0...	
Make sure that your saved checkpoint was also trained with GPU. If it was trained with CPU use -gpuid -1 for sampling as well	
Error: File lm_lstm_epoch5.05_1.1291.t7 does not exist. Are you sure you didn't forget to prepend cv/ ?	
/home/speed/torch/install/bin/luajit: cannot open <lm_lstm_epoch5.05_1.1291.t7> in mode r  at /home/speed/torch/pkg/torch/lib/TH/THDiskFile.c:670
stack traceback:
	[C]: at 0x7fc4a3a350e0
	[C]: in function 'DiskFile'
	/home/speed/torch/install/share/lua/5.1/torch/File.lua:405: in function 'load'
	sample.lua:86: in main chunk
	[C]: in function 'dofile'
	...peed/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00405e40
speed@speed-desktop:~/Downloads/char-rnn-master$ th sample.lua lm_lstm_epoch10.09_1.1358.t7 -gpuid -0 -length 10000
using CUDA on GPU 0...	
Make sure that your saved checkpoint was also trained with GPU. If it was trained with CPU use -gpuid -1 for sampling as well	
Error: File lm_lstm_epoch10.09_1.1358.t7 does not exist. Are you sure you didn't forget to prepend cv/ ?	
/home/speed/torch/install/bin/luajit: cannot open <lm_lstm_epoch10.09_1.1358.t7> in mode r  at /home/speed/torch/pkg/torch/lib/TH/THDiskFile.c:670
stack traceback:
	[C]: at 0x7f2e7071d0e0
	[C]: in function 'DiskFile'
	/home/speed/torch/install/share/lua/5.1/torch/File.lua:405: in function 'load'
	sample.lua:86: in main chunk
	[C]: in function 'dofile'
	...peed/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00405e40
speed@speed-desktop:~/Downloads/char-rnn-master$ th sample.lua lm_lstm_epoch10.09_1.1358.t7 -gpuid -0 -length 10000
using CUDA on GPU 0...	
Make sure that your saved checkpoint was also trained with GPU. If it was trained with CPU use -gpuid -1 for sampling as well	
Error: File lm_lstm_epoch10.09_1.1358.t7 does not exist. Are you sure you didn't forget to prepend cv/ ?	
/home/speed/torch/install/bin/luajit: cannot open <lm_lstm_epoch10.09_1.1358.t7> in mode r  at /home/speed/torch/pkg/torch/lib/TH/THDiskFile.c:670
stack traceback:
	[C]: at 0x7f013675d0e0
	[C]: in function 'DiskFile'
	/home/speed/torch/install/share/lua/5.1/torch/File.lua:405: in function 'load'
	sample.lua:86: in main chunk
	[C]: in function 'dofile'
	...peed/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00405e40
speed@speed-desktop:~/Downloads/char-rnn-master$ th sample.lua lm_lstm_epoch10.09_1.1358.t7 -gpuid -0 -length 10000
using CUDA on GPU 0...	
Make sure that your saved checkpoint was also trained with GPU. If it was trained with CPU use -gpuid -1 for sampling as well	
Error: File lm_lstm_epoch10.09_1.1358.t7 does not exist. Are you sure you didn't forget to prepend cv/ ?	
/home/speed/torch/install/bin/luajit: cannot open <lm_lstm_epoch10.09_1.1358.t7> in mode r  at /home/speed/torch/pkg/torch/lib/TH/THDiskFile.c:670
stack traceback:
	[C]: at 0x7f9cb7b150e0
	[C]: in function 'DiskFile'
	/home/speed/torch/install/share/lua/5.1/torch/File.lua:405: in function 'load'
	sample.lua:86: in main chunk
	[C]: in function 'dofile'
	...peed/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00405e40
speed@speed-desktop:~/Downloads/char-rnn-master$ 

System Running Code on: 

screenfetch
                          ./+o+-       speed@speed-desktop
                  yyyyy- -yyyyyy+      OS: ubuntu Ubuntu 14.04 trusty
               ://+//////-yyyyyyo      Kernel: x86_64 Linux 3.13.0-117-generic
           .++ .:/++++++/-.+sss/`      Uptime: 10m
         .:++o:  /++++++++/:--:/-      Packages: 4014
        o:+o+:++.`..```.-/oo+++++/     Shell: bash 4.3.11
       .:+o:+o/.          `+sssoo+/    Resolution: 1920x1080
  .++/+:+oo+o:`             /sssooo.   DE: Gnome 3.9.90
 /+++//+:`oo+o               /::--:.   WM: GNOME Shell
 \+/+o+++`o++o               ++////.   WM Theme: Adwaita
  .++.o+++oo+:`             /dddhhh.   GTK Theme: Radiance [GTK2/3]
       .+.o+oo:.          `oddhhhh+    Icon Theme: elementary-xfce-darker
        \+.++o+o``-````.:ohdhhhhh+     Font: Cantarell 11
         `:o+++ `ohhhhhhhhyo++os:      CPU: AMD FX-8350 Eight-Core @ 4GHz
           .o:`.syhhhhhhh/.oo++o`      GPU: GeForce GTX 750 Ti
               /osyyyyyyo++ooo+++/     RAM: 2150MiB / 24011MiB
                   ````` +oo+++o\:    
                          `oo++.
